module not working PLEASE HELP

**Hi this module use to work very well before.I just reinstalled the magic mirror and it is not working.
PLEASE HELP** 

**git clone https://github.com/paviro/MMM-PIR-Sensor.git
pi@raspberrypi:~/MagicMirror/modules $ cd MMM-PIR-Sensor
pi@raspberrypi:~/MagicMirror/modules/MMM-PIR-Sensor $ npm install**
npm WARN prefer global node-gyp@3.5.0 should be installed with -g

> epoll@0.1.20 install /home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/epoll
> node-gyp rebuild

make: Entering directory '/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/epoll/build'
  CXX(target) Release/obj.target/epoll/src/epoll.o
  SOLINK_MODULE(target) Release/obj.target/epoll.node
  COPY Release/epoll.node
make: Leaving directory '/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/epoll/build'

> Magic-Mirror-Module-PIR-Sensor@1.1.0 postinstall /home/pi/MagicMirror/modules/MMM-PIR-Sensor
> electron-rebuild -e ../../node_modules/electron-prebuilt

An unhandled error occurred inside electron-rebuild
Unable to find electron-prebuilt's version number, either install it or specify an explicit version

Error: Unable to find electron-prebuilt's version number, either install it or specify an explicit version
    at _callee$ (/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/electron-rebuild/lib/cli.js:74:17)
    at tryCatch (/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/regenerator-runtime/runtime.js:63:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/regenerator-runtime/runtime.js:337:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/regenerator-runtime/runtime.js:96:21)
    at GeneratorFunctionPrototype.tryCatcher (/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/bluebird/js/release/generators.js:97:49)
    at /home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/bluebird/js/release/generators.js:201:15
    at Object.<anonymous> (/home/pi/MagicMirror/modules/MMM-PIR-Sensor/node_modules/electron-rebuild/lib/cli.js:197:4)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)

npm ERR! Linux 4.4.34-v7+
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! Magic-Mirror-Module-PIR-Sensor@1.1.0 postinstall: `electron-rebuild -e ../../node_modules/electron-prebuilt`
npm ERR! Exit status 255
npm ERR!
npm ERR! Failed at the Magic-Mirror-Module-PIR-Sensor@1.1.0 postinstall script 'electron-rebuild -e ../../node_modules/electron-prebuilt'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the Magic-Mirror-Module-PIR-Sensor package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     electron-rebuild -e ../../node_modules/electron-prebuilt
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs Magic-Mirror-Module-PIR-Sensor
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls Magic-Mirror-Module-PIR-Sensor
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/pi/MagicMirror/modules/MMM-PIR-Sensor/npm-debug.log

-------------------------------------------



[npm-debug.zip](https://github.com/paviro/MMM-PIR-Sensor/files/698024/npm-debug.zip)

--- a/MMM-PIR-Sensor.js
+++ b/MMM-PIR-Sensor.js
@@ -9,6 +9,8 @@
 
 Module.register('MMM-PIR-Sensor',{
 
+	requiresVersion: "2.1.0",
+
 	defaults: {
 		sensorPIN: 22,
 		relayPIN: false,

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/paviro/MMM-PIR-Sensor#readme",
   "scripts": {
-    "postinstall": "node_modules/.bin/electron-rebuild -e ../../node_modules/electron-prebuilt"
+    "postinstall": "node_modules/.bin/electron-rebuild -e ../../node_modules/electron"
   },
   "dependencies": {
     "onoff": "latest"

